cmd/server-main.go

```diff
@@ -164,7 +179,7 @@ func init() {
 
 // serverMain handler called for 'minio server' command.
 func serverMain(ctx *cli.Context) {
-	if (!ctx.IsSet("sets") && !ctx.Args().Present()) || ctx.Args().First() == "help" {
+	if ctx.Args().First() == "help" && !endpointsPresent(ctx) {
```

Comment:
Needs to be `if ctx.Args().First() == "help" || !endpointsPresent(ctx) {`

Reply:
oh yes.

Reply:
actually no, if help is provided we should show help anyways.